plugin/core/windows.py

```diff
@@ -506,7 +506,7 @@ def log(self, message: str, params: Any) -> None:
 
         def run_on_async_worker_thread() -> None:
             nonlocal message
-            params_str = str(params)
+            params_str = repr(params) if isinstance(params, str) else str(params)
```

Comment:
It seems like we could use `repr` in all cases. At least objects and lists seem to render the same with both `repr` and `str`.

Reply:
Yeah I thought the same. The only difference between repr and str seems to be for floating point numbers, but they are very rare in LSP anyway.